frontend/device/android/device.lua

```diff
@@ -246,6 +246,11 @@ function Device:init()
         end,
     }
 
+    -- disable translation map for specific models
```

Comment:
That juts repeats what the code says. ;-) But alternatively you could link to `https://github.com/koreader/koreader/issues/12423` (or whichever) for background info.
```suggestion
```

Reply:
Adjusted the comment.